scheds/rust/scx_bpfland/src/main.rs

```diff
@@ -46,6 +48,69 @@ use scx_utils::UserExitInfo;
 
 const SCHEDULER_NAME: &'static str = "scx_bpfland";
 
+#[derive(Debug, Clone)]
+struct CpuMask {
+    mask: Vec<u64>,
+    num_bits: usize,
+}
+
+impl CpuMask {
+    pub fn from_mask(mask: Vec<u64>, num_bits: usize) -> Self {
+        Self { mask, num_bits }
+    }
+
+    pub fn is_cpu_set(&self, cpu: usize) -> bool {
+        if self.num_bits == 0 {
+            return true;
+        }
+        if cpu >= self.num_bits {
+            return false;
+        }
+        let idx = cpu / 64;
+        let bit = cpu % 64;
+        self.mask
+            .get(idx)
+            .map_or(false, |&val| val & (1 << bit) != 0)
+    }
+
+    pub fn from_str(hex_str: &str) -> Result<Self, std::num::ParseIntError> {
```

Comment:
in the future it might be nice to accept a `--cpu-list` similar to how [`taskset`](https://man7.org/linux/man-pages/man1/taskset.1.html) does it.

Reply:
@hodgesds good idea. I'll add that as a future change, it can be done all in Rust, so it should be trivial, thanks for the suggestion!